src/streamlink/plugins/tvrby.py

```diff
@@ -1,15 +1,16 @@
 from __future__ import print_function
 import re
 
+from streamlink import PluginError
 from streamlink.plugin import Plugin
 from streamlink.plugin.api import validate
 from streamlink.stream import HLSStream
 from streamlink.stream import RTMPStream
 
 
 class TVRBy(Plugin):
-    url_re = re.compile(r"""https?://(?:www\.)?tvr.by/televidenie/belarus""")
-    file_re = re.compile(r"""(?P<q1>["']?)file(?P=q1)\s*:\s*(?P<q2>["'])(?P<url>(?:http.+?m3u8.*?|rtmp://.*?))(?P=q2)""")
+    url_re = re.compile(r"""https?://(?:www\.)?tvr.by/plugines/online-tv-main\.php\?channel=tv&chan_id=""")
```

Comment:
All the normal URL still exist

example

- https://www.tvr.by/televidenie/belarus-1/
- https://www.tvr.by/televidenie/belarus-24/

you should search for `tvr.by/plugines/online-tv-main.php`on the normal URL,
instead of using it as the main URL